### PR TITLE
Backspace Button Changes

### PR DESCRIPTION
**Pull request by SanskrutiRP**
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by SanskrutiRP** [comment]
Hey @sairish2001  why it is creating conflicts I am not getting?
I didn't have modified anything in index.html only id attribute is added.

File: index.html [hunk 1]
```diff
@@ -1492,6 +1492,65 @@ <h5 class="modal-title">Rules for Roots of Quadratic Equation</h5>
                             <input type="tel" class="form__field" id="coffc" placeholder="Enter Number"
                                 oninput="rootquadratic();">
                         </div>
+
```
Comment (shrey27tri01) [inline index.html:1495]: Here is some extra code which you were trying to commit, which was causing the merge conflict. Can you have a look at it again?

**Review by shrey27tri01 — COMMENTED** [review]
@SanskrutiRP I have resolved the merge conflict. Please have a look at the inline review and remove the commented code.

File: index.html (hunk 1 above)
Comment (SanskrutiRP) [reply index.html:1495]: But this is not my code

Comment (shrey27tri01) [reply index.html:1495]: Okay, then, you can remove the commented part and I'll approve the PR.

Comment (shrey27tri01) [reply index.html]: I'm approving these changes as of now.

**Review by shrey27tri01 — APPROVED** [review]
Awesome work on this!

**Review by sairish2001 — COMMENTED** [review]
It is still creating conflicts, do update your code and make required changes

**Pull request merged**